From 37f1089c450fca16d06d586cf390a05642af25f0 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Mon, 4 Mar 2024 13:23:56 +0100
Subject: [PATCH 01/11] scripts/mk: only use ASCII characters

The policy recommends english, so french parenthesis must be replaced.
More generally, prudence recommends ASCII in Make scripts.
---
 scripts/mk/buildtools.mk | 2 +-
 scripts/mk/vendor.mk     | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/mk/buildtools.mk b/scripts/mk/buildtools.mk
index 933fdcfaa..7c6732210 100644
--- a/scripts/mk/buildtools.mk
+++ b/scripts/mk/buildtools.mk
@@ -20,7 +20,7 @@
 #   QMAKE: Qt build system generator (since dpkg 1.20.0).
 #
 # All the above variables have a counterpart variable for the build tool,
-# as in CC → CC_FOR_BUILD.
+# as in CC -> CC_FOR_BUILD.
 #
 # The variables are not exported by default. This can be changed by
 # defining DPKG_EXPORT_BUILDTOOLS.
diff --git a/scripts/mk/vendor.mk b/scripts/mk/vendor.mk
index f3241a57b..8bdaa235a 100644
--- a/scripts/mk/vendor.mk
+++ b/scripts/mk/vendor.mk
@@ -1,8 +1,8 @@
 # This Makefile fragment (since dpkg 1.16.1) defines the following
 # vendor-related variables:
 #
-#   DEB_VENDOR: output of «dpkg-vendor --query Vendor».
-#   DEB_PARENT_VENDOR: output of «dpkg-vendor --query Parent» (can be empty).
+#   DEB_VENDOR: output of "dpkg-vendor --query Vendor".
+#   DEB_PARENT_VENDOR: output of "dpkg-vendor --query Parent" (can be empty).
 #
 # This Makefile fragment also defines a set of "dpkg_vendor_derives_from"
 # macros that can be used to verify if the current vendor derives from
-- 
2.39.2

